### PR TITLE
Document dependency on 'pry' gem which shows correct usage

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,12 @@
 
 ## Installation
 
-To use this plugin, add the following like to the Gemfile in your Puppet code
+To use this plugin, add the following line to the `Gemfile` in your Puppet code
```
Comment (rnelson0) [inline README.md]: Might as well make `line` plural while we're at it :)

File: README.md [hunk 2]
```diff
@@ -4,11 +4,12 @@
...
 base and run `bundle install`.
 
 ```ruby
 gem 'puppet-lint-reference_on_declaration_outside_of_class-check'
+gem 'pry'
```
Comment (tuxmea) [inline README.md]: Please mention that pry is only used for debugging, not for using the lint extension.

Comment (ghoneycutt) [reply README.md]: @tuxmea I was not able to get the plugin to install until I added 'pry' to my Gemfile.

**Comment by ghoneycutt** [comment]
Please use the new squash feature when merging this.

**Comment by tuxmea** [comment]
Ah. a left over from debugging. Please rebase and remove the require pry statement. see #5

**Comment by juniorsysadmin** [comment]
Closing in favor of #6

**Comment by ghoneycutt** [comment]
Thanks!

**Pull request closed without merging**